enhanced/generate/main.go

```diff
@@ -0,0 +1,231 @@
+			"numVCPUs":           "The number of virtual CPUs for the DB instance.",
+			"timestamp":          "The time at which the metrics were taken.",
+			"uptime":             "The amount of time that the DB instance has been active.",
+			"version":            "The version of the OS metrics' stream JSON format.",
+		},
+		"cpuUtilization": {
+			"guest":  "The percentage of CPU in use by guest programs.",
+			"idle":   "The percentage of CPU that is idle.",
+			"irq":    "The percentage of CPU in use by software interrupts.",
+			"nice":   "The percentage of CPU in use by programs running at lowest priority.",
+			"steal":  "The percentage of CPU in use by other virtual machines.",
+			"system": "The percentage of CPU in use by the kernel.",
+			"total":  "The total percentage of the CPU in use. This value includes the nice value.",
+			"user":   "The percentage of CPU in use by user programs.",
+			"wait":   "The percentage of CPU unused while waiting for I/O access.",
+		},
+		"diskIO": {
+			"avgQueueLen":     "The number of requests waiting in the I/O device's queue. This metric is not available for Amazon Aurora.",
+			"avgReqSz":        "The average request size, in kilobytes. This metric is not available for Amazon Aurora.",
+			"await":           "The number of milliseconds required to respond to requests, including queue time and service time. This metric is not available for Amazon Aurora.",
+			"device":          "The identifier of the disk device in use. This metric is not available for Amazon Aurora.",
+			"readIOsPS":       "The number of read operations per second. This metric is not available for Amazon Aurora.",
+			"readKb":          "The total number of kilobytes read. This metric is not available for Amazon Aurora.",
+			"readKbPS":        "The number of kilobytes read per second. This metric is not available for Amazon Aurora.",
+			"rrqmPS":          "The number of merged read requests queued per second. This metric is not available for Amazon Aurora.",
+			"tps":             "The number of I/O transactions per second. This metric is not available for Amazon Aurora.",
+			"util":            "The percentage of CPU time during which requests were issued. This metric is not available for Amazon Aurora.",
+			"writeIOsPS":      "The number of write operations per second. This metric is not available for Amazon Aurora.",
+			"writeKb":         "The total number of kilobytes written. This metric is not available for Amazon Aurora.",
+			"writeKbPS":       "The number of kilobytes written per second. This metric is not available for Amazon Aurora.",
+			"wrqmPS":          "The number of merged write requests queued per second. This metric is not available for Amazon Aurora.",
+			"readLatency":     "TODO",
+			"writeLatency":    "TODO",
+			"writeThroughput": "TODO",
+			"readThroughput":  "TODO",
+			"diskQueueDepth":  "TODO",
```

Comment:
@AlekSi above metrics in vlimit are missing in http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_Monitoring.OS.html

Reply:
@delgod Does AWS has some advice for non-documented metrics? Are they stable? Can they be _suddenly_ removed?

Maybe we should just skip them?​

Reply:
I think we can scrape them, but shouldn't rely on.